---
source: compiler-core/src/type_/tests/errors.rs
assertion_line: 344
expression: "let add = fn(x, y) { x + y } 1 |> add(unknown)"

---
error: Unknown variable
  ┌─ /src/one/two.gleam:1:39
  │
1 │ let add = fn(x, y) { x + y } 1 |> add(unknown)
  │                                       ^^^^^^^ Did you mean `Error`?

The name `unknown` is not in scope here.

